qiskit/pulse/schedule/experiment_config.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Configurations for pulse experiments.
+"""
+import copy
+from typing import Dict, List
+from qiskit.pulse.channels import OutputChannel
+
+
+class ScheduleConfig:
+    """Configurations for user LO frequencies."""
+
```

Comment:
Can you add `Example::` how users can construct `ScheduleConfig` ? Specification of `user_lo_dic` is not intuitive.